store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreSchemaTest.java

```diff
@@ -135,6 +136,7 @@ public void shouldGetCorrectDefaultViewForAChosenGraphOperation() throws Operati
     }
 
     @Test
+    @Disabled //TODO FS OverlappingSchema review
```

Comment:
Is there a ticket for this/will it be a future alpha?

Reply:
@t92549  is on the case.